doc/content/ffmpeg_encoder.md

```diff
@@ -1,124 +1,162 @@
-- `%audio.copy` copies data without decoding or encoding it. This is great to avoid using the CPU but, in this case, the data cannot be processed through operators that modify it such as `fade.{in,out}` aor `smart_cross`. Also, all stream must agree on the same data format.
-- `%video(..)` is for options specific to the video codec. Unused options will raise an exception. Any option supported by `ffmpeg` can be passed here.
+- `<option_name>` can be any syntactically valid variable name or string. Strings are typically used when the option name is of the
+  form: `foo-bar`.
+- `%audio(...)` is for options specific to the audio codec. Unused options will raise an exception. Any option supported by `ffmpeg` can be
+  passed here. Streams encoded using `%audio` are using liquidsoap internal frame format and are fully handled on the liquidsoap side.
+- `%audio.raw(...)` behaves like `%audio` except that the audio data is kept as ffmpeg's internal format. This can avoid data copy and is
+  also the format required to use [ffmpeg filters](ffmpeg_filters.html).
+- `%audio.copy` copies data without decoding or encoding it. This is great to avoid using the CPU, but in this case, the data cannot be
+  processed with operators that modify it, such as `fade.{in,out}` or `smart_cross`. Also, all streams must agree on the same data format.
+- `%video(...)` is for options specific to the video codec. Unused options will raise an exception. Any option supported by `ffmpeg` can be
+  passed here.
 - `%video.raw` and `%video.copy` have the same meaning as their `%audio` counterpart.
-
-- Generic options are passed to audio, video and format (container) setup. Unused options will raise an exception. Any option supported by `ffmpeg` can be passed here.
+- Generic options are passed to audio, video and format (container) setup. Unused options will raise an exception. Any option supported
+  by `ffmpeg` can be passed here.
 
 ### HLS output
 
-The `%ffmpeg` encoder is the prime encoder for HLS output as it is the only one of our collection of encoder which can produce Mpeg-ts muxed data, which is required by most HLS clients.
+The `%ffmpeg` encoder is the prime encoder for HLS output as it is the only one of our collection of encoder which can produce Mpeg-ts muxed
+data, which is required by most HLS clients.
 
 ### File output
 
-Some encoding formats, for instance `mp4` require to rewing their stream and write a header after the fact, when encoding of the current track has finished. For historical reasons, such formats
-cannot be used with `output.file`. To remedy that, we have introduced the `output.url` operator. When using this operator, the encoder is fully in charge of the output file and can thus write headers
-after the fact. The `%ffmpeg` encoder is one such encoder that can be used with this operator.
+Some encoding formats, for instance `mp4` require to rewing their stream and write a header after the fact, when encoding of the current
```

Comment:
> Some encoding formats, for instance `mp4` require to rewing their stream and write a header after the fact, when encoding of the current track has finished.

I'm not sure what is *rewing*. @toots, could you please assist in deciphering this sentence?